website/docs/plugin/framework/migrating/attributes-blocks/default-values.mdx

```diff
@@ -1,19 +1,23 @@
 ---
 page_title: 'Attribute default values: Migrating from SDKv2 to the Framework'
 description: >-
-  Specify a default when the Terraform configuration does not supply a value for attributes.
+  Specify a default when the Terraform configuration does not supply a value for resource attributes.
   Migrate attribute defaults in SDKv2 to AttributePlanModifier in the Framework.
 ---
 
 # Default Values
 
-Default values set a value for an attribute when the Terraform configuration does not provide one. In SDKv2, default
-values are set via fields on an attribute's schema. In the Framework, you set default values via plan modification.
+Default values support is only available in the Framework for resources. Default values for data sources and
+providers should be handled within the [data source configure method](/terraform/plugin/framework/data-sources/configure#define-data-source-configure-method)
+or the [provider configure method](/terraform/plugin/framework/providers#configure-method), respectively.
```

Comment:
Defaults for data sources should be handled within the `Read` method for now. 😉 (Also minor grammar suggestions for active voice)

```suggestion
Default values support is only available in the Framework for resources. Handle default values for data source attributes within the [data source `Read` method](terraform/plugin/framework/data-sources#read-method) and default values for provider attributes within the [provider `Configure` method](/terraform/plugin/framework/providers#configure-method).
```